image/sigmiod_correction/sigmoid.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.